src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/TopNoteModalDialog.java

```diff
@@ -0,0 +1,55 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common;
+
+import com.google.common.base.Predicate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * This class should be used only for post details page. On this page only post can be approved or deleted, thus
+ * only on this page approve and remove modal dialogs will appear.
+ */
+public class TopNoteModalDialog extends BasePageObject {
+
+  private final long TIMEOUT = 10;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "TIMEOUT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)